ccloud/start-docker.sh

```diff
@@ -54,9 +54,11 @@ printf "Done\n\n"
 
 echo ====== Creating cloud topics users and pageviews and setting ACLs
 ccloud kafka topic create users
+ccloud kafka acl create --allow --service-account $serviceAccount --operation WRITE --topic users
+# Create pageviews topic because Replicator cannot automatically create the destination topic
+# when replication factor (RF) in the origin cluster < RF in the destination cluster
```

Comment:
Actually as of `5.5.0`, we've added support for the destination RF not being the same as the source RF. Specifying `dest.topic.replication.factor` sets that value as the RF for all topics created at the destination. More information here: 
https://docs.confluent.io/current/connect/kafka-connect-replicator/configuration_options.html#destination-topics

Reply:
@skaundinya15 thanks, I'm trying this out now but Replicator is unable to create the topic (note `WARN`):

```
connect-cloud                  | [2020-04-28 21:10:34,594] INFO Using user-specified replication factor of 3for creating destination topic pageviews (io.confluent.connect.replicator.ReplicatorSourceTask)
connect-cloud                  | [2020-04-28 21:10:34,661] INFO Creating topic pageviews in destination cluster with 6 partitions and replication factor 3 (io.confluent.connect.replicator.ReplicatorSourceTask)
connect-cloud                  | [2020-04-28 21:10:34,662] INFO Creating topic pageviews with Optional[6] partitions, replication factor Optional[3], and config {message.timestamp.type=CreateTime} (io.confluent.connect.replicator.util.ReplicatorAdminClient)
connect-cloud                  | [2020-04-28 21:10:35,286] WARN Failed to create or expand topic pageviews in the destination cluster. Will try again later. (io.confluent.connect.replicator.ReplicatorSourceTask)
```

If this is an ACL issue, which ACLs are required in CCloud to permit this?

Reply:
Actually I forgot to add a `CREATE`, let me try again...

Reply:
Ah okay, let me know if you run into any issues with the config.